CityOfDetroit/bloom
# 1460/listing finder translations

# Pull Request Template

## Issue Overview

This PR addresses #1460 and #1477.


## Description

This PR includes by the translations and updates seen [here](https://docs.google.com/spreadsheets/d/1glJRJ6znXsYJwF2AAI0w6xhN59ebcEMO8Qr0JU2mLnU/edit#gid=1450753116) **except for row 9** which is currently being discussed [here](https://exygy.slack.com/archives/C0203460K6Y/p1666395472098249) . In doing this, a bug came up around the new bordered fields which is #1474 and the homepage buttons in Spanish captured in #1477 .

## How Can This Be Tested/Reviewed?

This can be tested by setting the language on the homepage and then navigating through the listings finder to ensure that the translations are present. **Note**: Due to the way the translations are set within the finder, the language must be selected prior to filling out the questions (requires a refresh if done midway through which clears answers).
Additionally, the bottom three translations in the document are on the housing basics page.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
